### PR TITLE
Update the component READMEs

### PR DESCRIPTION
**Pull request by nandajavarma**

### CODE REVIEW

File: katalog/curator-s3/README.md [hunk 1]
```diff
@@ -1,32 +1,37 @@
 # Curator S3
 
+<!-- <KFD-DOCS> -->
+
 Curator helps you manage your Elasticsearch indices and snapshots via various
 operations like delete, snapshot, and shard allocation routing. It's mainly used
 to managed retention of your infrastructure logs to a given value.
 
+For `Curator` to work with `s3`, we will require an `AWS s3` compatible object
```
Comment (ralgozino) [inline katalog/curator-s3/README.md:9]: is s3 the right spelling or should it be S3?

**Review by ralgozino — CHANGES_REQUESTED** [review]
Can we add a minimal description to cerebro-s3 in the main readme?
Is `s3` the right spelling? I think it should be `S3`. It is used across several readmes.

File: katalog/curator-s3/README.md (hunk 1 above)
Comment (nandajavarma) [reply katalog/curator-s3/README.md:9]: fixed

Comment (nandajavarma) [reply katalog/curator-s3/README.md]: fixed it!

File: katalog/kibana/README.md [hunk 2]
```diff
@@ -45,6 +46,13 @@ kubectl port-forward svc/kibana 5601:5601 --namespace logging
 
 Kibana will be available on `http://127.0.0.1:5601` from your browser.
 
+Links
+
+[kibana-doc]: https://www.elastic.co/guide/en/kibana/7.16/index.html
+[kibana-github]:
```
Comment (ralgozino) [inline katalog/kibana/README.md:52]: missing link here

**Review by ralgozino — CHANGES_REQUESTED** [review]

File: katalog/kibana/README.md (hunk 2 above)
Comment (nandajavarma) [reply katalog/kibana/README.md]: sorry about that 🤦‍♀️  fixed now

**Review by ralgozino — APPROVED** [review]
lgtm!

**Pull request merged**